gematria/datasets/find_accessed_addrs.h

```diff
@@ -39,26 +39,40 @@ enum class VectorRegWidth : uint64_t {
 // bytes large, so that there will be no padding and calculating offsets by hand
 // is easy (as is required in our assembly prologue code).
 struct RawX64Regs {
-  VectorRegWidth max_vector_reg_width;
-  uint64_t uses_upper_vector_regs;
-  int64_t rax;
-  int64_t rbx;
-  int64_t rcx;
-  int64_t rdx;
-  int64_t rsi;
-  int64_t rdi;
-  int64_t rsp;
-  int64_t rbp;
-  int64_t r8;
-  int64_t r9;
-  int64_t r10;
-  int64_t r11;
-  int64_t r12;
-  int64_t r13;
-  int64_t r14;
-  int64_t r15;
+  VectorRegWidth max_vector_reg_width;  // offset 0x0
+  // If true, the code uses at least one of the 16 extra vector registers
+  // defined in AVX-512. This is interpreted in combination with the max width.
+  // For example, if max_vector_reg_width is XMM and uses_upper_vector_regs is
+  // true, then the code uses XMM0-XMM31 but no YMM or ZMM registers.
+  //
+  // If this is false, then the latter 16 elements of vector_regs are unset and
+  // should be ignored.
+  uint64_t uses_upper_vector_regs;  // offset 0x8
+  // If true, the code uses at least one of the 16 extra general purpose
+  // registers defined in APX.
+  //
+  // If this is false, then the elements of apx_regs are unset and should be
+  // ignored.
+  uint64_t uses_apx_regs;  // offset 0x10
```

Comment:
As discussed internally: Let's add a note that this is `int64_t` and not `bool` to make the layout of the struct simpler.